apps/taxes/src/modules/taxjar/logs/ui/taxjar-logs.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Text } from "@saleor/macaw-ui/next";
+import { TaxJarLogsTable, RefreshLogsButton } from "./taxjar-logs-table";
+
+export const TaxJarLogs = () => {
```

Comment:
how is this component unique to taxjar? Why don't you just create Logs/LogsTable and pass logs in props?

then you can have a simple wrapper component that will resolve logs from metadata based on config id?